### PR TITLE
Document monthly social meeting

### PR DESCRIPTION
**Pull request by graingert**
- [ ] Closes #xxxx
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Review by jrbourbeau — COMMENTED** [review]
Thanks for the PR @graingert! I'm curious what prompted this change as the zoom link should still be correct for the monthly community meeting

File: docs/source/support.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Conversation happens in the following places:
     10:00 US Central Time in `this video meeting <https://us06web.zoom.us/j/87619866741?pwd=S2RxMlRKcnVvakt4NHZoS1cwOGZoZz09>`_.
     Meeting notes are available in
     `this Google doc <https://docs.google.com/document/d/1UqNAP87a56ERH_xkQsS5Q_0PKYybd5Lj2WANy_hRzI0/edit>`_.
-
+#.  **Weekly social meeting** happens every Thursday at 08:00 US Central Time in `this video meeting <https://whereby.com/dask-dev>`_.
```
Comment (graingert) [inline docs/source/support.rst:37]:
```suggestion
#.  **Weekly social meeting** happens every Thursday at 08:00 US Central Time in `this video meeting <https://whereby.com/dask-dev>`_.

```

**Comment by ncclementi** [comment]
In the last Dask community meeting, it was decided that "We’re going to switch to once-monthly, US-EU friendly.", and that we would switch that in the calendar. 

@james I believe you have access to the calendar?

**Comment by jrbourbeau** [comment]
Yep, this is on my TODO list

**Review by jrbourbeau — APPROVED** [review]
Thanks for the reminder @ncclementi. I just updated the text here to reflect the discussion during the last monthly meeting. Note the Google calendar invite has _not yet been updated_. I'll merge this PR once the calendar is up to date.

**Pull request merged**